sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_providers/local_provider.py

```diff
@@ -64,34 +64,37 @@ def encrypt(self, algorithm, plaintext, iv=None):
         # type: (EncryptionAlgorithm, bytes, Optional[bytes]) -> EncryptResult
         self._raise_if_unsupported(KeyOperation.encrypt, algorithm)
         ciphertext = self._internal_key.encrypt(plaintext, algorithm=algorithm.value, iv=iv)
-        return EncryptResult(key_id=self._key.kid, algorithm=algorithm, ciphertext=ciphertext, iv=iv)
+        return EncryptResult(
+            key_id=self._key.kid, algorithm=algorithm, ciphertext=ciphertext, iv=iv)  # type: ignore[attr-defined]
```

Comment:
nit: for each of these, we should put the trailing parenthesis on its own line
```suggestion
        return EncryptResult(
            key_id=self._key.kid, algorithm=algorithm, ciphertext=ciphertext, iv=iv
        )  # type: ignore[attr-defined]
```